### ISSUE TITLE
Some random noise on the left upper corner generated images

### ISSUE THREAD

**Issue by kirill-ionkin** [facebookresearch/vsait#7]
Hello, thks for your great work!
I trained VSAIT on different datasets and see some problems, which are not described on paper or any issues on github.

At first,(when training on datasets: Cityscapes with Sim200K):

at this generated images i see something like black hole next to the wheel disk of the machine
![3349429](https://user-images.githubusercontent.com/56236621/231686688-a695b9b2-6b3b-44d4-a20c-7a22a3038d24.jpg)

above the back of the car i see also something like black hole, where model generated something bad 
![3382653](https://user-images.githubusercontent.com/56236621/231686697-9beda5c3-453b-4f6a-9fa9-28e4856b450c.jpg)

In my opinion - as if the model is generating an image that has some area that it is not sure about

For, examle, other dataset(Summer to Winter):

![summer_zone1__0](https://user-images.githubusercontent.com/56236621/231688127-6768139c-ecc9-4949-b7be-b5fefb847c9f.jpg)
![summer_zone1__158](https://user-images.githubusercontent.com/56236621/231688134-2c09e6c3-aab5-4a90-b999-57b38554c1f5.jpg)
both images have some random white noise on the left upper corner and i actually don't understand how to fix this problems?

Did you see something like this during training and create model architecture? How to fix this problem?
Tnks you for your attention)

**Comment by jdtheiss** [comment]
Thanks for your interest in our work! This is interesting; I don't think I've observed the first examples (i.e., "black hole"), but I believe I've seen the latter examples before. I don't recall the exact context, but it might have something to do with large differences in pixel intensities between domains and/or some other artifact related to the adversarial training. Here are a few ideas to try:

- Lower learning rate and/or try different beta parameters/weight decay (edit in the [model config's OPTIMIZERS node](https://github.com/facebookresearch/vsait/blob/main/configs/vsait.yaml#L24))
- Try a different [coefficient](https://github.com/facebookresearch/vsait/blob/main/configs/vsait.yaml#L22) for the "VSA" loss
- Separately normalize with domain-specific image statistics prior to feature extraction by adding the following to the [model config's FEATURE_EXTRACTOR node](https://github.com/facebookresearch/vsait/blob/main/configs/vsait.yaml#L8):
```
RESCALE:
    source:
        - {SOURCE_MEAN}
        - {SOURCE_STD}
    target:
        - {TARGET_MEAN}
        - {TARGET_STD}
```
- Change the LSH parameters, e.g., for denser sampling by changing the [stride in the model config's LSH node](https://github.com/facebookresearch/vsait/blob/main/configs/vsait.yaml#L19)
- Customize the [mapping network definition](https://github.com/facebookresearch/vsait/blob/main/models/iit.py#L121) by adding a `MAP_NET_KWARGS` node [under the model config's TASK_MODEL node](https://github.com/facebookresearch/vsait/blob/main/configs/vsait.yaml#L1), e.g.:
```
MAP_NET_KWARGS:
    n_channels: {list of conv layer out channels}
    kernel_size: {kernel size used for each conv layer}
    padding: {padding used for each conv layer}
```

Hope these suggestions can help you resolve the issues!

**Comment by jdtheiss** [comment]
I hope those suggestions were helpful, but feel free to reopen the issue if you have further questions.
